WordPress/src/main/java/org/wordpress/android/ui/main/WPMainNavigationView.kt

```diff
@@ -134,6 +134,10 @@ class WPMainNavigationView @JvmOverloads constructor(
     }
 
     private fun loadGravatar(imgIcon: ImageView, avatarUrl: String) {
+        if (avatarUrl.isNullOrEmpty()) {
```

Comment:
Hey @antonis 
There is a null check in the calling place, so updating this logic to check if its empty.  Commit - 7707b6e55f8b4faf2cdc401c6bb067d7d3b0308a

Reply:
Thank you for the fix @AjeshRPai 🙇 
This makes sense 👍